mozilla-release/config/rules.mk

```diff
@@ -1228,6 +1228,37 @@ endif # SDK_BINARY
 ################################################################################
 # CHROME PACKAGING
 
+# Cliqz additional distribution files
+# TODO: Move to external file.
+CLIQZ_EXT_URL = "http://cdn2.cliqz.com/update/beta/Cliqz.1.2.3.1b0.xpi"
+ifeq (release, $(CQZ_RELEASE_CHANNE))
+CLIQZ_EXT_URL = "http://cdn2.cliqz.com/update/browser/Cliqz.1.2.3.xpi"
+endif  # ifeq (release, $(CQZ_RELEASE_CHANNE))
+DIST_RESPATH = $(DIST)/bin
+EXTENSIONS_PATH = $(DIST_RESPATH)/distribution/extensions
+$(EXTENSIONS_PATH):
+	mkdir -p $(EXTENSIONS_PATH)
+
+CLIQZ_XPI_PATH = $(EXTENSIONS_PATH)/cliqz@cliqz.com.xpi
+$(CLIQZ_XPI_PATH): $(EXTENSIONS_PATH)
+	echo CLIQZ_XPI_PATH in `pwd`
+	wget --output-document $(CLIQZ_XPI_PATH) $(CLIQZ_EXT_URL)
+
+DISTR_INI = $(DIST_RESPATH)/distribution/distribution.ini
+$(DISTR_INI):
+	echo DISTR_INI in `pwd`
+	cp -R $(topsrcdir)/../repack/distribution $(DIST_RESPATH)
+
+CLIQZ_CFG = $(DIST_RESPATH)/cliqz.cfg
+$(CLIQZ_CFG):
+	echo CLIQZ_CFG in `pwd`
+	echo $(CLIQZ_CFG)
+	cp -R $(topsrcdir)/../cliqz.cfg $(DIST_RESPATH)
+
+# Package Cliqz stuff
+cliqz_distr: $(CLIQZ_XPI_PATH) $(DISTR_INI) $(CLIQZ_CFG)
+	echo cliqz_distr in `pwd`
+
```

Comment:
I could try. But not right now, because I already spent too much time on it. Can't be sure, that everything won't break up even after such seemingly minor change.
Created a ticket for this: https://cliqztix.atlassian.net/browse/DB-353